waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/metrics/MonitoredAspectTest.java

```diff
@@ -109,6 +115,23 @@ public void monitorSuccesses() throws Throwable {
     assertThat(rs.timer().count(), is(1L));
```

Comment:
Please also add asserts to this integration test: https://github.com/ExpediaGroup/waggle-dance/blob/main/waggle-dance-integration-tests/src/test/java/com/hotels/bdp/waggledance/WaggleDanceIntegrationTest.java#L403